pom.xml

```diff
@@ -53,6 +53,14 @@ tools.jackson.core.*;version=${project.version}
 
     <!-- for Reproducible Builds -->
     <project.build.outputTimestamp>2022-11-27T00:00:00Z</project.build.outputTimestamp>
+
+    <!-- dependency and plugin versions -->
```

Comment:
These are to be upgraded in parent pom:

https://github.com/FasterXML/oss-parent/

so could you please remove?